code/modules/reagents/reagent_containers/food/snacks.dm

```diff
@@ -573,6 +573,71 @@
 			src.name = "Frosted Jelly Donut"
 			reagents.add_reagent("sprinkles", 2)
 
+/obj/item/weapon/reagent_containers/food/snacks/donut/stat_buff
+	name = "Masterpiece Donut"
+	desc = "The taste you will never forgot."
```

Comment:
Should be: "The taste you will never forget."

Reply:
Done